modules/common-jpa-impl/src/main/java/org/opencastproject/serviceregistry/impl/jpa/ServiceRegistrationJpaImpl.java

```diff
@@ -66,7 +66,7 @@
                 + "WHERE job.processorServiceRegistration.online=true and job.processorServiceRegistration.active=true and job.processorServiceRegistration.hostRegistration.maintenanceMode=false "
                 + "AND job.status in :statuses "
                 + "AND job.creatorServiceRegistration.serviceType != :workflow_type "
-                + "GROUP BY job.processorServiceRegistration.hostRegistration.baseUrl, job.status"),
+                + "GROUP BY job.processorServiceRegistration.hostRegistration.baseUrl, job.status, job.processorServiceRegistration.hostRegistration.maxLoad"),
```

Comment:
This seems unrelated to fixing Postgres?

Reply:
It's related. PostgreSQL will not allow thew previous query …and if you think about it, it does make sense that having multiple values for `maxload` would be problematic in the query. I guess MariaDB is just ignoring additional values.

Reply:
mysql will also not allow that query (before the change) in certain modes: https://dev.mysql.com/doc/refman/5.7/en/group-by-handling.html

Reply:
Huh, I suppose it's technically possible that such a value could exist, but our business logic precludes that from actually happening.  Interesting that one driver objects.